src/pseudopeople/configuration/interface.py

```diff
@@ -14,26 +14,25 @@ def get_config(dataset_name: str = None, overrides: Union[Path, str, Dict] = Non
     """
     Function that returns the pseudopeople configuration,
     including all default values.
-    If :code:`dataset_name` is None (the default), the returned dictionary has exactly
+    If :code:`dataset_name` is None (the default), the returned dictionary includes
+    the configuration for all datasets. If a dataset name is supplied, only that
+    dataset's configuration is returned. In both cases, the returned dictionary has exactly
     the structure described on the :ref:`Configuration page <configuration_main>`.
-    If a dataset name is supplied, only returns the part of the configuration for that
-    dataset, i.e. :code:`get_config(dataset_name)` is equivalent to
-    :code:`get_config()[dataset_name]`.
 
     To get the default probability of nonresponse in the Decennial Census dataset:
 
     .. code-block:: pycon
 
         >>> import pseudopeople as psp
-        >>> psp.get_config('decennial_census')['row_noise']['do_not_respond']
+        >>> psp.get_config('decennial_census')['decennial_census']['row_noise']['do_not_respond']
```

Comment:
It now seems redundant to pass the dataset argument to the function here, as you could just call `psp.get_config()['decennial_census']['row_noise']['do_not_respond']`. I wonder if it would be confusing to users to pass it in both places -- personally, this would make me wonder wonder if that was necessary for some reason.

cc @aflaxman @zmbc

Reply:
agree, should we remove the option to pass a dataset argument?

Reply:
I don't think we should get rid of it. It's useful, especially now, because you can call `get_config(dataset_name)` instead of having to do `{dataset_name: get_config()[dataset_name]}`. I just think it's confusing to pass the dataset name in this particular example because it's unnecessary.

Reply:
But maybe it would be worth coming up with an example where you _do_ want to call `get_config(dataset_name)` so we can show what that looks like?